runtime/compiler/env/VMJ9.cpp

```diff
@@ -8350,6 +8350,29 @@ TR_J9VM::getROMMethodFromRAMMethod(J9Method *ramMethod)
    return J9_ROM_METHOD_FROM_RAM_METHOD(ramMethod);
    }
 
+bool 
+TR_J9VM::noMultipleConcreteClasses(List<TR_PersistentClassInfo>* subClasses)
+   {
+   TR::Compilation *comp = TR::comp();
```

Comment:
You could use `TR::Compilation *comp = _compInfoPT->getCompilation();`
or pass the comp object as a parameter to make it clear this function can only be executed during compilation

Reply:
Use the same for JITServer implementation